src/App.tsx

```diff
@@ -1,15 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function contentCheck(value: string, query: string) {
```

Comment:
The function's name should start with a verb (so you could easily tell what your method actually do).

```suggestion
function checkContent(value: string, query: string) {
```